tools/table_annovar/README

```diff
@@ -19,6 +20,7 @@ ANNOVAR needs to be installed manually in the following way:
 	
 	iii) edit the tool-data/annovar.loc file to reflect location of humandb folder
 
-2) add the annovar scripts convert2annovar.pl and table_annovar.pl to your Galaxy user's path
+2) set the SCRIPT_PATH variable in the associated env.sh file to directory where the perl scripts (annotate_variation.pl and convert2annovar.pl) are installed.
```

Comment:
The scripts should be on PATH, or the user should extend PATH in the env.sh file, this would make SCRIPT_PATH unnecessary ...